cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenAccount.js

```diff
@@ -0,0 +1,46 @@
+var checkoutConfiguration = window.Configuration;
+checkoutConfiguration.paymentMethodsConfiguration = {
+    card: {
+        enableStoreDetails: false,
+        hasHolderName: true,
+        onBrand: function (brandObject) {
+            $('#cardType').val(brandObject.brand);
+        },
+        onFieldValid: function (data) {
+            if (data.endDigits) {
+                maskedCardNumber = MASKED_CC_PREFIX + data.endDigits;
+                $("#cardNumber").val(maskedCardNumber);
```

Comment:
This will make sure that the maskedCardNumber is displayed on Summary page, confirmation email etc. For the default behaviour in SFRA, we will have to set the cardNumber.